Dockerfile.server

```diff
@@ -2,8 +2,10 @@
 
 FROM node:16
 
-RUN apt-get update
-RUN apt-get --yes install vim
+RUN apt-get update \
+    && apt-get install -y \
+    vim \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Thanks for the change. Please could you add a comment to explain why we delete the list?

Reply:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

The main idea here is that you should always run apt-get update when you install packages in docker .. so you assume the cached lists are stale .. if you assume they're stale .. why even bother taking up space in the image with a stale copy of them .. since if you ever do an apt-get install you will need to fetch them again anyways ..

Reply:
On that same note .. should we even be installing vim into the image? It seems like something we don't actually need in a final image .. do we?

Reply:
I was thinking about that too, we can remove it actually